tests/storage/test_storage.py

```diff
@@ -262,12 +262,13 @@ def test_write_read_tiledbsoma(storage):
     artifact_soma = save_tiledbsoma_experiment(
         [test_file],
         description="test tiledbsoma",
-        key="scrna/my-big-dataset",  # can also be None, but that's trivial
+        key="scrna/my-big-dataset.tiledbsoma",  # can also be None, but that's trivial
```

Comment:
@Koncopd, I'm very surprised that this inconsistent suffix in the key was possible before. Let me know if you think this should have been the case.

It clashes with all other conventions which is why it was now raised as an error.

Reply:
I think you are correct, this was a mistake actually.